### ISSUE TITLE
Move example screenshots from https://eclipse.dev/nebula/ to this repo

### ISSUE THREAD

**Issue by vogella** [EclipseNebula/nebula#536]
For see example of the nebula widgets I have to go to https://eclipse.dev/nebula/, would it be possible to move these examples to this repo for example as EXAMPLES.md?

**Comment by wimjongman** [comment]
Good idea. Can you create a PR?

**Comment by lcaron** [comment]
Sorry for my very late answer : I postpone this issue to next release.

**Comment by vogella** [comment]
I think I can handle that... I'm currently migrating some plaform wikis, once this is done I should have scripts for such migrations.

**Comment by wimjongman** [comment]
@vogella does that offer still count?

**Comment by vogella** [comment]
I can give it next week a try. Is that OK?

**Comment by vogella** [comment]
Once the move has merged, I suggest to format https://eclipse.dev/nebula/ to the new page

**Comment by wimjongman** [comment]
Oh, you added it to the main repository? Maybe not a bad idea but I also started a Github wiki. https://github.com/EclipseNebula/nebula/wiki

Maybe we should not have a Github wiki at all but just use the docs folder? What do you all think?

**Comment by vogella** [comment]
I personally prefer docs as we have them under version control and can get PRs.

Anything in the wiki which I should move to the docs folder?

**Comment by vogella** [comment]
> Once the move has merged, I suggest to format https://eclipse.dev/nebula/ to the new page

@wimjongman can you do that?

**Comment by laeubi** [comment]
A github pages page is also version controlled, also you can have PRs for a wiki page.

**Comment by wimjongman** [comment]
> Once the move has merged, I suggest to format https://eclipse.dev/nebula/ to the new page

What do you mean by this?

**Comment by vogella** [comment]
> > Once the move has merged, I suggest to format https://eclipse.dev/nebula/ to the new page
> 
> What do you mean by this?

Retire https://eclipse.dev/nebula/ by replacing it's content with: This page has moved to new_link. Or a HTTP redirect if possible
